themes/openy_themes/openy_rose/templates/page/page--footer.html.html.twig

```diff
@@ -64,7 +64,7 @@
         {% if footer_logo %}
           {% set background_image = " style='background-image: url(" ~ file_url(footer_logo) ~ ")'" %}
         {% endif %}
-        <a href="{{ front_page }}" class="footer__brand brand" title="{{ "YMCA"|t }}"{{ background_image|raw }}></a>
+        <a href="{{ front_page }}" class="footer__brand brand" title="{{ "Site-logo"|t }}"{{ background_image|raw }}></a>
```

Comment:
This change affects only footer for 3rd party services http://ci.openymca.org/build1805/footer.html.

We need to cover 3 other use cases:
- Footer in global template https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/page/page.html.twig#L157
- Footer on camp page https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/page/page--node--camp.html.twig#L45
- Landing page associated with Camp https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/page/page--node--landing-page--has-camp.html.twig#L45